components/public-api-server/pkg/webhooks/stripe_test.go

```diff
@@ -22,9 +23,10 @@ import (
 
 // https://stripe.com/docs/api/events/types
 const (
-	invoiceUpdatedEventType   = "invoice.updated"
-	invoiceFinalizedEventType = "invoice.finalized"
-	customerCreatedEventType  = "customer.created"
+	invoiceUpdatedEventType     = "invoice.updated"
+	invoiceFinalizedEventType   = "invoice.finalized"
+	customerCreatedEventType    = "customer.created"
+	customerSubscriptionDeleted = "customer.subscription.deleted"
```

Comment:
Might make sense to move these as consts in the `stripe.go` file and re-use in tests